server/app/views/admin/programs/ProgramFormBuilder.java

```diff
@@ -143,10 +146,18 @@ private FormTag buildProgramForm(
       formTag.with(
           FieldWithLabel.checkbox()
               .setFieldName("isCommonIntakeForm")
-              .setLabelText("Set program as common intake")
+              .setLabelText("Set program as pre-screener")
+              .addStyleClass("border-none")
               .setValue("true")
               .setChecked(isCommonIntakeForm)
-              .getCheckboxTag());
+              .getCheckboxTag()
+              .with(
+                  span(ViewUtils.makeSvgToolTip(
+                          "Setting a program as ‘pre-screener’ will reconfigure and pin this"
```

Comment:
nit: unclear at first glance what's being reconfigured. maybe something like "will reconfigure the programs page. This card will be pinned to the top..."

Reply:
Updated it to match the copy in Miriam's UX copy doc.